### PR TITLE
Implemented basic (non saving) signup routes

### PR DESCRIPTION
**Pull request by Davidde94**

### CODE REVIEW

File: Sources/SpartaLib/Facades/HuntersFacade.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//
+//  HuntersFacade.swift
+//  SpartaLib
+//
+//  Created by David Evans on 25/07/2018.
+//
+
```
Comment (RyanMKrol) [inline Sources/SpartaLib/Facades/HuntersFacade.swift]: Song

File: Sources/SpartaLib/Facades/HuntersFacade.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    func handleSignupEmailRequest(_ request: RouterRequest, response: RouterResponse) {
+        self.executeRequest(request, response: response) { () -> SimpleResponse in
+            
+            let onboarding = try request.codable(for: HunterEmailOnboardingInfo.self)
+            let hunter = try Hunter(onboarding)
+            
+            let response = GenericResponse<Hunter>(
+                message: "Hunter created",
+                restResponseCode: .created,
+                payload: hunter
```
Comment (RyanMKrol) [inline Sources/SpartaLib/Facades/HuntersFacade.swift]: I don't know what the API should return for this. We should probably discuss and document expected responses and well as requests.

File: Sources/SpartaLib/Routes.swift [hunk 3]
```diff
@@ -0,0 +1,15 @@
+//
+//  Routes.swift
+//  SpartaLib
+//
+//  Created by David Evans on 23/07/2018.
+//
+
```
Comment (RyanMKrol) [inline Sources/SpartaLib/Routes.swift]: song

File: Sources/SpartaLib/Routes.swift [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import Foundation
+
+class Routes {
+    
+    static let CreateUserThirdParty = "/user/signup/thirdparty"
```
Comment (RyanMKrol) [inline Sources/SpartaLib/Routes.swift]: These routes are different from what we documented, maybe update the README to be a single source of truth on this

File: Sources/SpartaLib/Server.swift [hunk 5]
```diff
@@ -0,0 +1,43 @@
+//
+//  Server.swift
+//  SpartaLib
+//
+//  Created by David Evans on 23/07/2018.
+//
+
```
Comment (RyanMKrol) [inline Sources/SpartaLib/Server.swift]: song

File: Sources/SpartaLib/Facades/HuntersFacade.swift [hunk 6]
```diff
@@ -0,0 +1,48 @@
+//
+//  HuntersFacade.swift
+//  SpartaLib
+//
+//  Created by David Evans on 25/07/2018.
+//
+
+import Foundation
+import Kitura
```
Comment (RyanMKrol) [inline Sources/SpartaLib/Facades/HuntersFacade.swift]: Madness has a dependency on Kitura, is this import needed?

Comment (Davidde94) [reply Sources/SpartaLib/Facades/HuntersFacade.swift]: `import` in Swift is equivalent to `include` in C++, this just makes the package available to that file.

File: Sources/SpartaLib/Facades/HuntersFacade.swift (hunk 2 above)
Comment (Davidde94) [reply Sources/SpartaLib/Facades/HuntersFacade.swift]: #jazzyTime https://github.com/realm/jazzy

File: Sources/SpartaLib/Routes.swift (hunk 4 above)
Comment (Davidde94) [reply Sources/SpartaLib/Routes.swift]: Ah I actually made this before the note, I'll update

**Pull request merged**